src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
According to the checklist, you should use fallback fonts. This is a good practice to ensure that your website will still look presentable even if the main font doesn't load. You can add a generic font family as a fallback, like this: `font-family: Roboto, Arial, sans-serif;`